src/components/dashboard/sidebar/SubsidiesSummary.jsx

```diff
@@ -23,16 +24,57 @@ import { COURSE_STATUSES } from '../../../constants';
 import { getStatusMetadata } from '../data/utils';
 import useExpirationMetadata from '../../budget-expiry-notification/data/hooks/useExpirationMetadata';
 
+const SearchCoursesCta = ({
+  ctaButtonVariant,
+  showSearchCoursesCta,
+  isProgramProgressPage,
+  disableSearch,
+  slug,
+  className,
+}) => {
+  if (disableSearch && !showSearchCoursesCta && !isProgramProgressPage) {
```

Comment:
also here, shouldn't this be more like this?
```suggestion
  if (disableSearch || !showSearchCoursesCta || isProgramProgressPage) {
```

Reply:
Yep, actually working through a test that handles this and validating the behavior 👍🏽  great catch.